test/DebugInfo/OpenCL100/DebugInfoSubrange.ll

```diff
@@ -14,8 +14,8 @@
 ; CHECK-SPIRV: Constant [[#TypeInt64Id]] [[#NegativeCount:]] 4294967295 4294967295
 
 ; CHECK-SPIRV: [[#DbgFuncId:]] [[#]] DebugFunction [[#FuncNameId]]
+; CHECK-SPIRV: [[#]] [[#DbgLocVarId:]] [[#]] DebugLocalVariable [[#VarNameId]] [[#]] [[#]] [[#]] [[#]] [[#DbgFuncId]]
```

Comment:
Basically here is one of the effects of https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2033/files#diff-ce5c478b99f08cea7d78a07829f628f520b4ab8746eb64c37874ad4f88fbccabR266 - DbgTemplateId is changed to DbgFuncId which is actually the correct behavior